pom.xml

```diff
@@ -119,7 +119,7 @@
 			<dependency>
 				<groupId>life.qbic</groupId>
 				<artifactId>data-model-lib</artifactId>
-				<version>2.12.0-SNAPSHOT</version>
+				<version>2.12.1</version>
```

Comment:
you updated dml here but did not indicate it in the CL, please add this :)

Reply:
I think it is mentioned in 1.1.1 😉